src/context/store.tsx

```diff
@@ -73,7 +76,8 @@ const StoreContextProvider = ({
         customerGroup: context?.customerGroup ?? '',
         userViewHistory: context?.userViewHistory ?? [],
       },
-      apiUrl: environmentType?.toLowerCase() === 'testing' ? TEST_URL : API_URL,
+      promoTilesDataPath: '/drafts/kevin/promo-tiles.json',
```

Comment:
i think this should just be passed in above alongside `apiUrl` - i'll make the change and get this merged